lib/config/guiconfiguration.go

```diff
@@ -59,6 +61,14 @@ func (c GUIConfiguration) Address() string {
 	return c.RawAddress
 }
 
+func (c GUIConfiguration) Permissions() os.FileMode {
```

Comment:
The function name permissions is too generic. If we decide to add ldap permissioning to the gui, this name being taken would be grim.

Reply:
Done.